### DISCUSSION TITLE
Difference between V0.41 and V0.42 PCB

### DISCUSSION THREAD

**Discussion by vk5ln** [frntc/Sidekick64#30] [General]
Hi,
Great to see the development of a Raspberry Pi Zero SideKick64, very cool!

I've had a look at both the V0.41 (the PCB I have) and the V0.42 and can't see obvious difference between the two. What was changed? 

Is the change easy to implement on the V0.41 PCB?

**Comment by frntc** [comment]
(If I remember correctly) it was just a pull-up resistor which was not connected properly on 0.41. In principle only the detection of the SID-wire in the menu was affected and this was catched by the Sidekick-code. So no need to modify anything! (In case I overlook something, let me know)

**Reply by vk5ln** [reply]
Very good. Thanks for the info.

Now I look closer in that area I can see the added feedthroughs and track on the back of the PCB for the SID-wire pull up resistor.
It doesn't look too hard to add a small wire link... but if the code handles the missing resistor then I might not bother. :-)

Keep up the great work. Love my SideKick64
